### PR TITLE
Add help command

### PR DESCRIPTION
**Pull request by rharwo**

### CODE REVIEW

**Review by wx-03 — COMMENTED** [review]
lgtm, but perhaps we can explain that the fields in curly brackets are required and the fields in square brackets are optional

File: docs/UserGuide.md [hunk 1]
```diff
@@ -329,6 +329,11 @@ Run the [`list spendings`](#listing-all-spendings) command to check the index of
     ____________________________________________________________
 ```
 
+### Getting user instructions
+The `help` command allows the user to see a quick summary of the commands.
+
+**Format:** `help`
+
```
Comment (TPH777) [inline docs/UserGuide.md:336]: Maybe can consider showing the example output?

**Review by TPH777 — COMMENTED** [review]
Lgtm! But i think its better to add the output of help in UG, although all the details are already in the UG.

File: docs/UserGuide.md (hunk 1 above)
Comment (rharwo) [reply docs/UserGuide.md]: Fixed!

**Pull request merged**